src/main/java/movie/storage/Main.java

```diff
@@ -26,7 +28,7 @@ public static void main(String[] args) {
         Movie secondMovie = new Movie();
         secondMovie.setTitle("Suits");
         movieService.add(secondMovie);
-        movieService.getAll().forEach(System.out::println);
+        movieService.getAll().forEach(movie -> logger.info(movie + "is added"));
```

Comment:
are you adding or getting objects?

Reply:
The add is displayed for the user, read only to display the added data